WindowsAppRuntime.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.32106.194
+# Visual Studio Version 17
+VisualStudioVersion = 17.2.32616.157
```

Comment:
Repo has 6 .sln

* C:\source\repos\windowsappsdk2\WindowsAppRuntime.sln
* C:\source\repos\windowsappsdk2\dev\MRTCore\mrt\MrtCore.sln
* C:\source\repos\windowsappsdk2\dev\VSIX\WindowsAppSDK.Extension.sln
* C:\source\repos\windowsappsdk2\installer\WindowsAppRuntimeInstall.sln
* C:\source\repos\windowsappsdk2\installer\test\CreateTestPackages\CreateTestPackages.sln
* C:\source\repos\windowsappsdk2\test\WindowsAppSDK.Helix.Test\WindowsAppSDK.Helix.Test.sln

The VSIX .sln already refers to v17 though a different one. That should be fixed?

The other 5 were v16. They're all updated to 17.2.blahblah? matching this line?